packages/project-config/package.json

```diff
@@ -18,7 +18,7 @@
     "dist"
   ],
   "scripts": {
-    "build": "yarn node ./build.js && run build:types",
```

Comment:
This one is `build.ts`, most others have been `build.mts`. What's the heuristics for picking the file extension?

Reply:
This package is ESM so `.ts` files are ESM already here, they don't need the `m` prefix